android/src/main/java/org/strongswan/android/logic/CharonVpnService.java

```diff
@@ -438,10 +444,13 @@ private Notification buildNotification(boolean publicVersion) {
             builder.setPublicVersion(buildNotification(true));
         }
 
-//        Intent intent = new Intent(getApplicationContext(), MainActivity.class);
```

Comment:
Can we remove this code? If we are not gonna use it

Reply:
Yup this is a removal

Reply:
This is a removal of the existing comment